packages/amplify_datastore/android/build.gradle

```diff
@@ -52,8 +54,8 @@ android {
 
 dependencies {
     implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
-    implementation "com.amplifyframework:aws-datastore:1.1.3"
-    implementation "com.amplifyframework:aws-api-appsync:1.1.3"
+    implementation "com.amplifyframework:aws-datastore:main"
+    implementation "com.amplifyframework:aws-api-appsync:main"
```

Comment:
I wonder if there is any way that we can centralize all of these `main` references, around the codebase, into a single `amplify-flutter`-side variables somewhere.

Reply:
I don't think we will be keeping these reference changes for the final release - this is just temporary.

Reply:
The problem is independent of whatever this string happens to say. `1.1.3` should be updated to the latest version sooner than later. It'd be advantageous to do that change in one location. Also, the developer use case of writing `main` would be facilitated.

Reply:
Right that makes sense.  Sounds good to consider unified version name variable for a future PR.